### PR TITLE
repeatability of experiments

### PR DESCRIPTION
**Pull request by gueguenster**
I did not progress more than installation:

- requirements.txt needs to have pinned versions
- installation steps for YOLOv3 needs to be developed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,20 +21,30 @@ Contact: [chaabane@colostate.edu](mailto:chaabane@colostate.edu). Any questions
...
-conda create -n Objects_Geolocalization
+git clone https://github.com/MedChaabane/Static_Objects_Geolocalization.git
+cd Static_Objects_Geolocalization/
+conda create -y -n Objects_Geolocalization python=3.7
 conda activate Objects_Geolocalization
-conda install pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
-cd ${OBJECTS_GEOLOCALIZATION}
-pip install -r requirements.txt
+conda install -y pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
+pip install -r requirements.txt  # need to pin versions in this file
```
Comment (gueguenster) [inline README.md:32]: That steps fail for me, could you pin all the versions in requirements.txt? @MedChaabane

File: README.md [hunk 2]
```diff
@@ -21,20 +21,30 @@ Contact: [chaabane@colostate.edu](mailto:chaabane@colostate.edu). Any questions
...
 ```
 
 ## Extract TLG Data set
 In order to extract TLG dataset:
 * The nuScenes dataset should first downloaded from their [official webpage](https://www.nuscenes.org).
 * Install [YOLO v3](https://github.com/eriklindernoren/PyTorch-YOLOv3) and download the [pretrained model](https://github.com/eriklindernoren/PyTorch-YOLOv3/blob/master/weights/download_weights.sh).
+```
+conda activate Objects_Geolocalization
+mkdir YOLO
```
Comment (gueguenster) [inline README.md:41]: There are steps missing here, that I tried to recover. But I was not successful....
I cannot import YOLO.PyTorchYOLOv3.*, in subsequent scripts

File: extract_TLG_dataset.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import argparse
 import os
+import math
```
Comment (gueguenster) [inline extract_TLG_dataset.py:3]: seems missing that import

File: README.md (hunk 1 above)
Comment (gueguenster) [reply README.md:32]: could you export the conda environment:
```
conda env export > environment.yml
```
and place that at the root of this github repo.

File: README.md [hunk 4]
```diff
@@ -21,20 +21,30 @@ Contact: [chaabane@colostate.edu](mailto:chaabane@colostate.edu). Any questions
...
-conda install pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
-cd ${OBJECTS_GEOLOCALIZATION}
-pip install -r requirements.txt
+conda install -y pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
+pip install -r requirements.txt  # need to pin versions in this file
 ```
 
 ## Extract TLG Data set
 In order to extract TLG dataset:
 * The nuScenes dataset should first downloaded from their [official webpage](https://www.nuscenes.org).
```
Comment (gueguenster) [inline README.md:37]: could you indicate which dataset to downlowd, maybe having a picture could help. There are many links on the download page.

File: README.md (hunk 1 above)
Comment (MedChaabane) [reply README.md:32]: yes that is done.

File: README.md (hunk 4 above)
Comment (MedChaabane) [reply README.md:37]: Full dataset and map expansion should be downloaded. I specified that now in the readme.

**Pull request merged**